.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,9 +14,11 @@ Contributors guide: https://github.com/electron/electron/blob/master/CONTRIBUTIN
 - [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
 - [ ] relevant documentation is changed or added
 - [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
+- [ ] PR release notes describe the change in a way relevant to app-developers
 
 
 #### Release Notes
-<!-- Used to describe release notes for future release versions. Use `no-notes` to indicate no user-facing changes. -->
 
-Notes: <!-- One-line Change Summary Here-->
\ No newline at end of file
+<!-- The note that you write here will be communicated to Electron app developers in the release notes when this change is released. Write something short but descriptive (one sentence to a couple paragraphs, if appropriate) that explains what, if anything, this change means for a developer building an app with Electron. Use `no-notes` for things like chores, internal fixes, and the like. -->
```

Comment:
```suggestion
 <!-- The note that you write here will be communicated to Electron app developers in the release notes when this change is released. Write something short but descriptive (one line, but it can be a long one) that explains what this change means for a developer building an app with Electron. If there's nothing to communicate to app developers, use the special token `no-notes` to indicate that no release note should be included relating to this change. -->
```

Reply:
It has paragraph support (even though it doesn't work perfectly atm, but @ckerr is fixing bugs around this).

Reply:
I'm not sure how far we want to go with that. Yes if someone writes a paragraph of text, e.g. something that is equivalent except to a long line except for the user typing linefeeds in the PR form, I don't see the harm in supporting that.

However with PRs like https://github.com/electron/electron/pull/15976 where you have an actual bulletpoint list in the release note... I'm not in favor of supporting that. If each Note can have its own formatting, that's going to make the release notes look cluttered.

We may want to reconsider: what problem are we solving with multi-paragraph release notes?

Reply:
also worth noting that the notes in #15976 are not appropriate for the release notes form, either (they don't describe what the fix does in app-developer-relevant terms)

Reply:
I think that, ideally, fewer PRs would have release notes, and the notes would be a little more fleshed out. For complex changes, especially new features, I'd be happy to see a slightly longer release note.

Reply:
That's reasonable. I'll see what I can do to make Notes like https://github.com/electron/electron/pull/15976 look better.